features/demo/src/demo.js

```diff
@@ -28,20 +28,19 @@ window.addEventListener("DOMContentLoaded", () => {
             });
         });
 
-    (function() {
+    (function () {
         let wastedMemory = "";
 
         function wasteMemory(mb) {
             let oneMbString = "";
-            for (i = 0; i < 1024 * 1024; i++)
+            for (let i = 0; i < 1024 * 1024; i++)
                 oneMbString += Math.floor(Math.random() * 10);
-            for (i = 0; i < mb; i++)
-                wastedMemory += oneMbString;
+            for (let i = 0; i < mb; i++) wastedMemory += oneMbString;
         }
 
         function updateWastedMemory() {
-            document.querySelector(".wasted-memory")
-                .innerText = wastedMemory.length / 1024 / 1024;
+            document.querySelector(".wasted-memory").innerText =
+                wastedMemory.length / 1024 / 1024;
```

Comment:
The changes in this file is that lint fix you made on `main` - might do something weird if you attempt to merge it.

Reply:
Um, right. Will merge from main again just in case.